src/redux/trainers/thunks.js

```diff
@@ -32,3 +38,41 @@ export const updateTrainer = async (dispatch, id, body) => {
     dispatch(updateTrainerError(true));
   }
 };
+
+export const getTrainers = async (dispatch) => {
+  try {
+    const reponse = await fetch(`${process.env.REACT_APP_API_URL}/api/trainer`);
+    const data = await reponse.json();
+    const listTrainers = data.data;
+    console.log(listTrainers);
+    if (listTrainers.length === 0) {
+      dispatch(getTrainersPending(true));
+      console.log(true);
```

Comment:
This console.log should be deleted.

Reply:
I'm working in it. Thanks for notifying.